easybuild/easyconfigs/l/libgpuarray/libgpuarray-0.7.6-fosscuda-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CMakeMake'
+
+name = 'libgpuarray'
+version = '0.7.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://deeplearning.net/software/libgpuarray/'
+description = "Library to manipulate tensors on the GPU."
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+
+source_urls = ['https://github.com/Theano/libgpuarray/releases/download/v%(version)s/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['887b6433a30282cb002117da89b05812c770fd9469f93950ff3866ddd02bfc64']
+
+builddependencies = [('CMake', '3.11.4')]
+dependencies = [('Python', '2.7.15')]
+
+exts_defaultclass = 'PythonPackage'
+exts_list = [
+    (name, version, {
+        'buildcmd': 'build_ext',
+        'modulename': 'pygpu',
+        'source_tmpl': '%(name)s-%(version)s.tar.gz',
```

Comment:
@akesandgren Please use `SOURCE_TAR_GZ`?

Reply:
But that doesn't work in exts_list does it?

Reply:
`SOURCE_TAR_GZ` and `'%(name)s-%(version)s.tar.gz'` are exactly the same, if one works so will the other

Reply:
I thought I had problems using those in exts_list, but i'll try...